src/test/java/LoginTest.java

```diff
@@ -16,6 +17,12 @@ public void successLogin() throws InterruptedException {
         ContactsPage contactsPage = new ContactsPage(app.driver);
         sleep(1000);
```

Comment:
why sleep is here?

Reply:
I need change method sleep to isDisplayed.

Reply:
Doesn’t it work without `sleep`?